pkg/helm/handlers/handler_test.go

```diff
@@ -719,7 +748,7 @@ func TestHelmHandlers_Index(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			var request *http.Request
 
-			request = httptest.NewRequest(http.MethodGet, "/api/helm/charts/index.yaml", strings.NewReader(""))
+			request = httptest.NewRequest(http.MethodGet, "/api/helm/charts/index.yaml"+tt.urlQuery, strings.NewReader(""))
```

Comment:
we should also have a test where `onlyCompatible` is send from client and the value is `false`.

Reply:
added